gluefactory/datasets/oxford_paris_mini_1view_jpldd.py

```diff
@@ -257,14 +299,28 @@ def _read_groundtruth(self, image_folder_path, shape: int = None) -> dict:
         kp_file = image_folder_path / "keypoints.npy"
         kps_file = image_folder_path / "keypoint_scores.npy"
         dlsd_kp_gt_file = self.dlsd_kp_gt_folder / image_folder_path.name / "base_image.npy"
+        dlsd_line_gt_file = self.dlsd_line_gt_folder / image_folder_path.name / "line_endpoints.npy"
 
         #Load Line GT
         # Load pickle file for DF max and min values
         with open(image_folder_path / "values.pkl", "rb") as f:
             values = pickle.load(f)
 
         # Load DF
+        use_binary_gt = self.conf.load_features.line_gt.use_binary_gt
+        thickness = self.conf.load_features.line_gt.interpolated_line_thickness
+
         df_img = read_image(image_folder_path / "df.jpg", True)
+
+        if use_binary_gt:
+            # read dlsd line gt
+            df_img = np.zeros((df_img.shape[0], df_img.shape[1]), dtype=np.float32)
+            line_endpoints = np.load(dlsd_line_gt_file)
+            for line in line_endpoints:
+                start_point = (int(round(line[0][0])), int(round(line[0][1])))
+                end_point = (int(round(line[1][0])), int(round(line[1][1]))) 
+                cv2.line(df_img, start_point, end_point, color=255, thickness=thickness)
+
         df_img = df_img.astype(np.float32) / 255.0
         df_img *= values["max_df"]
```

Comment:
this shouldn't be done when using binary line heatmap gt as this multiplication could lead to the values being >1 which is forbidden for a probabilistic map.